pkgs/tools/misc/tldr/default.nix

```diff
@@ -11,9 +11,14 @@ stdenv.mkDerivation rec {
     sha256 = "10ylpiqc06p0qpma72vwksd7hd107s0vlx9c6s9rz4vc3i274lb6";
   };
 
-  buildInputs = [ curl clang libzip ];
+  buildInputs = [ git curl clang libzip ];
```

Comment:
Git should be left out. It is just used for getting the tag which won't work here because fetchFromGitHub leaves out the .git directory.

Reply:
Thanks for the pointer 😄